### ISSUE TITLE
[BUG] Bottom Actions not showing sometime

### ISSUE THREAD

**Issue by Shajeel-Afzal** [sarbagyastha/youtube_player_flutter#135]
I am trying to show CurrentPosition, ProgressBar, RemainingDuration, and FullScreenButton with the video but these are not showing for some reason. I have two main list items and if I Navigate to the screen which contains the youtube player from the 1st list item, it shows the controls, but it does not show the controls if I navigate from the second list item. 

**To Reproduce**

**Expected behavior**
I am expecting it to show the controls every time based on my bottomActions widgets. 

 - Device: Samsung S8
 - OS: Android

here is my code: 

```
YoutubePlayer(
            controller: _controller,
            aspectRatio: 16 / 9,
            showVideoProgressIndicator: false,
            controlsTimeOut: Duration(seconds: 2),
            progressIndicatorColor: Colors.red,
            bottomActions: [
              CurrentPosition(),
              SizedBox(width: 10.0),
              ProgressBar(isExpanded: true),
              SizedBox(width: 10.0),
              RemainingDuration(),
              FullScreenButton(),
            ],
            onReady: () {
              _isPlayerReady = true;
            },
          ),
```

**Comment by MatthewManageMy** [comment]
I am getting this issue as well, on a range of Android devices.

It seems to happen after a user exits from the videos full screen mode, every time.

For now I have disabled full screen mode, but would appreciate a fix.

**Comment by rafaelmaeuer** [comment]
I experienced the issue too, happening most of the time in fullscreen mode

**Comment by sarbagyastha** [comment]
Should fix with v7.0.0. Feel free to reopen if the issue persists.

**Comment by csof3cen** [comment]
I just installed version 8.1.2 and still facing the same issue. Bottom actions are not showing, even when not using full-screen.
